webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -1099,7 +1099,7 @@ export function getNeededAtMentionedUsernamesAndGroups(state: GlobalState, posts
 
                 if (attachment.fields) {
                     for (const field of attachment.fields) {
-                        findNeededUsernamesAndGroups(field.value);
+                        findNeededUsernamesAndGroups(String(field.value));
```

Comment:
Instead of casting this to a string, could we check `typeof field.value === 'string'` before doing this? I feel like there was a similar bug to this caused by someone passing a non-string value here, and we don't need to look for at-mentions in a string anyway